launchUI.py

```diff
@@ -669,7 +669,7 @@ def leave(event):
 def clickTile(ID):
     widgets["fuConfigPannel"].config(text='Tile '+str(ID)+' functional units')
     widgets["xbarConfigPannel"].config(text='Tile '+str(ID)+' crossbar outgoing links')
-    widgets["xbarConfigPannel"].grid(columnspan=4, row=7, column=0,rowspan=2)
+    widgets["xbarConfigPannel"].grid(columnspan=4, row=7, column=0,rowspan=2,sticky="nsew")

```

Comment:
Can you add a comment here about the meaning of "nsew"?

Reply:
Yes. In this sentence "nsew" means that each xbar component will fill the corresponding grid space. Similar usage is seen in the subsequent code.